api/v1alpha2/machine_types.go

```diff
@@ -37,9 +37,9 @@ const (
 
 // MachineSpec defines the desired state of Machine
 type MachineSpec struct {
-	// ObjectMeta will autopopulate the Node created. Use this to
-	// indicate what labels, annotations, name prefix, etc., should be used
-	// when creating the Node.
+	// ObjectMeta specifies the metadata to associate with a Machine
+	// when using MachineSet or MachineDeployment.
+	// You can only specify Labels and Annotations, all other fields are ignored.
```

Comment:
Where do we use the Labels and Annotations?

Reply:
https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/machineset_controller.go#L366-L367

Reply:
Those 2 fields are from MachineTemplateSpec's embedded ObjectMeta, not this one.

Reply:
omg

Reply:
Then this field isn't used anywhere

Reply:
Can we mark as deprecated?

Reply:
If I remove this field entirely, it doesn't appear to be used anywhere except in some test code.

Reply:
I changed the test code, and marked it as deprecated in the comment, given that we can't remove it because it'd be a breaking change